### ISSUE TITLE
I want to create and save drawings. How do I do that?

### ISSUE THREAD

**Issue by ggzzzzz628** [ryan-brazeal-ufl/RPyLIDAR#2]
This project is great!
You can see the real-time radar map data

I want to create and save drawings. How do I do that?

**Comment by agentscout101** [comment]
I agree, I want to be able to store the data pulled (distance and angle) so I can then create a map using encoders as well
